### DISCUSSION TITLE
How to properly use automatic translation from shared project memory

### DISCUSSION THREAD

**Discussion by auronen** [WeblateOrg/weblate#9432] [Q&A] [answered]
Hello,
I have various translation projects that share the same base set of strings and I would like to use the automatic translation from shared memory. This works amazingly, when the projects have the same base/source language, but when the projects have different base/source language, it does not work at all (0 strings translated).

Is this an intended feature? Is there a way to configure the behavior?

**Comment by nijel** [comment, marked as answer]
The translation memory stores source/translation pairs, it cannot transpose to other translations or do reverse lookups.
